### PR TITLE
fix: lower the freqency of available disk space alerts

### PR DESCRIPTION
**Pull request by lemeurherve**
by increasing the duration of the measurement.

### CODE REVIEW

File: datadog-monitors.tf [hunk 1]
```diff
@@ -24,7 +24,7 @@ resource "datadog_monitor" "disk_space" {
   type    = "query alert"
   message = "@pagerduty"
 
-  query = "avg(last_5m):exclude_null(avg:system.disk.free{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) / exclude_null(avg:system.disk.total{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) * 100 < 10"
+  query = "avg(last_10m):exclude_null(avg:system.disk.free{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) / exclude_null(avg:system.disk.total{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) * 100 < 10"
```
Comment (lemeurherve) [inline datadog-monitors.tf:27]:
```suggestion
  query = "avg(last_15m):exclude_null(avg:system.disk.free{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) / exclude_null(avg:system.disk.total{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) * 100 < 10"
```
10 min should be enough for most builds, but increasing it to 15 min shouldn't make a lot of difference while letting plenty of time to alert before filling disks.

**Comment by dduportal** [comment]
What is the goal of this change? it's not clear

**Comment by lemeurherve** [comment]
> What is the goal of this change? it's not clear

As we received numerous alerts for the privatek8s' linuxpool disk space since #168 and as they all were resolved by themselves in less than 10 minutes, I though of increasing the average measurement duration to lower the frequency of these alerts to decrease alert fatigue.

An alternative would be to increase a bit more privatek8s linuxpool disk size, like what have been done in https://github.com/jenkins-infra/azure/pull/329

**Comment by dduportal** [comment]
Thanks for the explanation, it makes sense.

I've opened https://github.com/jenkins-infra/azure/pull/330 to increase a bit more the disk size, as the 80% threshold on disks implies performance changes: better to use bigger disk and ensure that compute time is decreased (costs less)

**Comment by lemeurherve** [comment]
@dduportal while I was commenting you've opened a PR with the alternative ^^
https://github.com/jenkins-infra/azure/pull/330

Closing this one.

**Comment by dduportal** [comment]
Thanks for looking on this topic ❤️

**Pull request closed without merging**